### PR TITLE
Document new `dd_hostname` field for fluentbit integration

### PR DESCRIPTION
**Pull request by jszwedko**
### What does this PR do? What is the motivation?

Documents the new `dd_hostname` field added to the fluentbit integration: https://github.com/fluent/fluent-bit/pull/8988

### Merge instructions

- [ ] Please merge after reviewing

Waiting for fluentbit release.

### CODE REVIEW

File: content/en/integrations/fluentbit.md [hunk 1]
```diff
@@ -38,17 +38,18 @@ Before you begin, you need to have a [Datadog account][3], a [Datadog API key][4
...
+| dd_message_key | _Recommended_ - Set the attribute to use to store your log message.                                                                                                                                                                                                                                                         |                                                                             |
+| dd_tags        | _Optional_ - The [tags][10] you want to assign to your logs in Datadog.                                                                                                                                                                                                                                                     |                                                                             |
+| dd_hostname    | _Optional_ - The host that emitted logs should be associated with. If unset, Datadog will expect the host to be set as one of [the standard host attributes][12].
```
Comment (hestonhoffman) [inline content/en/integrations/fluentbit.md:51]:
```suggestion
| dd_hostname    | _Optional_ - The host that emitted logs should be associated with. If unset, Datadog expects the host to be set as one of [the standard host attributes][12].
```

**Review by hestonhoffman — APPROVED** [review]
Small edit. I'll commit and merge

**Comment by hestonhoffman** [comment]
@jszwedko I noticed you didn't check the box in the PR description. Am I good to merge this one?

**Comment by jszwedko** [comment]
Thanks @hestonhoffman . Note I think we may want to hold off on merging this until the next fluentbit release since the option is currently unreleased. I'll drop a comment when this is ready to be merged.

**Comment by hestonhoffman** [comment]
@jszwedko Ah good call. Thanks! Let us know when it's ready to go and we'll merge it in

**Comment by jszwedko** [comment]
@hestonhoffman this is good to merge now since the change [has been released](https://github.com/fluent/fluent-bit/releases/tag/v3.1.0).

**Pull request merged**